library/src/main/java/com/novoda/downloadmanager/DownloadManagerBuilder.java

```diff
@@ -234,8 +259,8 @@ public void onServiceDisconnected(ComponentName name) {
                 fileOperations,
                 downloadsBatchPersistence,
                 downloadsFilePersistence,
-                notificationChannelCreator,
-                notificationCreator,
+                notificationConfig,
+                notificationConfig,
```

Comment:
`NotificationConfig` implements both of the old interfaces so can be passed like this. Reason for doing this is just to keep the pull request size a bit smaller.

Reply:
Will this change in a follow-up PR?

Reply:
Yes, planning to change it in the next PR or the one after.